scripts/migrations/migrations.sh

```diff
@@ -21,8 +20,17 @@ function run_migrations() {
     VERSION="0.9.0"
   fi
 
-  rm -r -f $INFRA_PATH/euclid-development-environment
-  rm -f $ROOT_PATH/old-euclid.json
+  euclid_field_version=$(jq -r '.deploy.ansible.monitoring // empty' "$ROOT_PATH/euclid.json")
```

Comment:
Can't this use the euclid.json version field instead of this `deploy.ansible.monitoring` check?